CHANGELOG.md

```diff
@@ -4,6 +4,15 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 7.22.0
```

Comment:
h: this should just be a patch, the core `feat` is completely internal to us.

Reply:
We're extending the API in two places. Adding an argument to `recordDroppedEvent` is definitely a change that forces a minor and the remix typing changes are also technically a minor.

A minor is just correct here.

Reply:
`recordDroppedEvent` is not completely internal to us. Users can pass that hook.

Reply:
Those are both fair points, but the general likelyhood of an external user using `recordDroppedEvent` is extremely low. I wouldn't try to follow semvar to that precisely, I would optimize for making sure that upgrades are easy as possible, and so patches >> minor. This also applies to the remix typing changes, it's just a bug fix.